node/src/chain/acceptor.rs

```diff
@@ -190,7 +192,9 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             .await
             .update(|t| t.set_register(blk.header()))?;
 
-        *provisioners_list = self.vm.read().await.get_provisioners()?;
+        provisioners_list.replace(self.vm.read().await.get_provisioners()?);
+        provisioners_list.forget_prev();
```

Comment:
why do we forget prev?
also, maybe it would make sense to use a different method than `replace` if we don't want to keep `prev`

Reply:
We forget the previous, because it's not different from the current